### ISSUE TITLE
Generate custom handwriting?

### ISSUE THREAD

**Issue by SB2020-eye** [ankanbhunia/Handwriting-Transformers#11]
Could this be used to reproduce one particular handwriting style? (That is, one not in the IAM database -- such as my own handwriting, or the handwriting of a famous historical figure.)

If so, could you please walk me through how to do that? I would be very appreciative.

**Comment by ankanbhunia** [comment]
I haven't tried with custom handwritting. However, you can refer to the demo.ipynb file. Cross-dataset evaluation (ie. CVL trained model tested on IAM test set) is shown in the script. The results were satisfactory. I guess, the model will perform well on the custom handwritting as well. 

You can use the function `page_val = model._generate_page()` (refer to demo.ipynb) and input your custom handwriting (resized and padded to [1, 15, 32, 192]; 15=no number of words, 32=height, 192=padded width). Don't forget to do necessary preprocessing.

**Comment by SB2020-eye** [comment]
Thanks so much for replying.

I imagine that you answered my question very well. Regrettably, I am too much of a novice to understand how exactly to do what you are describing. If you're willing to elaborate and walk through the steps like someone would for a beginner, that would be immensely appreciated. If not, I know that one's time is valuable and would totally understand. 😃

**Comment by sah1lga1kwad** [comment]
> Thanks so much for replying.
> 
> I imagine that you answered my question very well. Regrettably, I am too much of a novice to understand how exactly to do what you are describing. If you're willing to elaborate and walk through the steps like someone would for a beginner, that would be immensely appreciated. If not, I know that one's time is valuable and would totally understand. 😃

https://github.com/Shambu-K/handwriting-generator-model

Use this: This the implementation of custom handwriting using this Github Repo
If you need help executing, message here.

**Comment by ankanbhunia** [comment]
A new notebook file is added for custom handwriting (https://colab.research.google.com/github/ankanbhunia/Handwriting-Transformers/blob/main/demo_custom_handwriting.ipynb).

**Comment by mikecummings34** [comment]
I am getting an index our of range error when trying to use a custom dataset.  
I think it has something to do with only having one writer in my dataset? 
The data set has 1 writer, with about 5000 examples. 
 When I print the length of the enumerated  torch.dataloader, it shows length of 0.
 If I get length of TextDatasetval.__get__item(0) I get a length of 7.  
So I think my dataset is ok and formatted correctly.  Any clarification would be appreciated.
